deploy/staging/cronjob.yaml

```diff
@@ -13,10 +13,10 @@ spec:
       template:
         spec:
           containers:
-          - name: prison-visits-booking-staff-pvb-metrics-refresh
-            image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/prison-visits-booking/prison-visits-staff:latest
-            imagePullPolicy: Always
-            command: ['sh', '-c', "bundle exec rake pvb:metrics:refresh"]
+            - name: prison-visits-booking-staff-pvb-metrics refresh
```

Comment:
Is the missing hyphen in the name deliberate?

Reply:
👍